documentatie/ontwikkelproces/pull-requests.md

```diff
@@ -26,6 +26,7 @@ Let minstens op:
 - scope van de wijzigingen
 - kwaliteit van de code
 - consistentie met omliggende code
+- imports en de resulterende afhankelijkheden
```

Comment:
I think the goal is to pay more attention to the high-level structure of the code. I'm also not completely sure how to write this down concisely, but maybe:

```suggestion
- imports en de resulterende afhankelijkheden
- indeling van code in bestanden/componenten/modules
```

Reply:
Replaced the "imports en ..." bullet with your suggestion.